alignak/objects/config.py

```diff
@@ -1103,13 +1103,14 @@ def read_config_buf(self, buf):  # pylint: disable=R0912
 
             line_nb += 1
             # Remove comments
-            line = split_semicolon(line)[0].strip()
```

Comment:
So we consider semicolon on continuation line as legit char ? This affect all file parsing here, carful